workflows/prognostic_c48_run/runtime/transformers/fv3fit.py

```diff
@@ -60,4 +61,4 @@ def partial_fit(self, inputs: State, state: State):
 
     @property
     def input_variables(self) -> Iterable[Hashable]:
-        return self.model.input_variables
+        return list(set(self.model.input_variables) | set(self.config.variables))
```

Comment:
ensure we have both the variables needed for ML prediction (i.e. the ML inputs) and the state variables that we are applying the predictions to.